examples/cifar10/cifar10-real-world/virtualenv/min-requirements.txt

```diff
@@ -0,0 +1,5 @@
+nvflare>=2.2.1rc6
+torch
+torchvision
+tensorboard
+tenseal==0.3.12
```

Comment:
Using nvflare[he], it will still install the old tenseal version `tenseal==0.3.0`. I tested the newer version here which is preferred. Keeping as is for now.